string language_code
---
string sentence
